mastering-plone/views_3.rst

```diff
@@ -365,10 +365,13 @@ Add this simple table to :file:`templates/talklistview.pt`:
 
 Again we use ``class="listing"`` to give the table a nice style.
 
-There are some some things that need explanation:
+There are some things that need explanation:
 
 :samp:`tal:define="talks python:view.talks()"`
-    This defines the variable `talks`. We do thins since we reuse it later and don't want to call the same method twice. Since TAL's path expressions for the lookup of values in dictionaries is the same as for the attributes of objects and methods of classes we can write :samp:`view/talks` as we could :samp:`view/someattribute`. Handy but sometimes irritating since from looking at the page template alone we often have no way of knowing if something is an attribute, a method or the value of a dict.
+    This defines the variable `talks`.
+    We do this since we reuse it later and don't want to call the same method twice.
+    Since TAL's path expressions for the lookup of values in dictionaries is the same as for the attributes of objects and methods of classes we can write :samp:`view/talks` as we could :samp:`view/someattribute`.
+    Handy but sometimes irritating since from looking at the page template alone we often have no way of knowing if something is an attribute, a method or the value of a dict.
 
 :samp:`tal:repeat="talk talks"`
     This iterates over the list of dictionaries returned by the view. Each :py:obj:`talk` is one of the dictionaries that are returned by this method.
```

Comment:
@svx the Mastering Plone training generally contains a lot of rule violations (obviously because it's older than the style guide). I'm trying to fix things while I'm editing content, but will leave other locations untouched (imo changing them makes it harder to review the non whitespace changes).

I'd suggest using dedicated PRs for fixing style guide violations.

Reply:
**Disclaimer !!**

The following is not meant to be personal or insulting or something, all of you are totally awesome ! 

Yeah, I know, that is also why I was not saying anything about headers, etc, etc. :)

Last year, when I started to work on the trainings, the "mastering" stood untouched, because working on the others took way longer, because there was way more to do (still not 100%) finished.
Because of that I decided last year to do not touch "mastering" because I do not want to end up  half finished.

Because of this confusing layout for the user.

It is not "only" about the style-guide.
It is also about the quality of the ``rst``. 
Things like *proper* ``rst`` and taking care about white spaces, we could work on that already because it would make improving things later a bit less work :) 

At the moment, in general I am a bit worried about, that people push to master without **proper** review.

I have no idea, if people really do not care or not seeing it or maybe not knowing it :)

It is not only about that your docs/training is technical OK, it is also about consistent wording, style, readability, etc, etc, etc.

The whole idea about the training is that people can also follow the trainings without a trainer :)
It seems to me, that sometimes this gets forgotten :)

The visual factor, how it looks like as HTML is important, too :)

That is, why I was trying to say, that I am a bit concerned about the whole "push direct to master" thing ....

In general we do not do that with code, so why do that with docs, as many many times already said **docs = code**.

If you have awesome code (software) but **not that awesome docs** your product is **broken**.

Sorry all for the rant, but sometimes I get a bit frustrated :)

Reply:
@pbauer 
I am sorry if my previous post was not clear enough.
Like I wrote, it is/was not my intention to insult, hurt or do something like that.
It is also not my intention to rant about the generally bad state.
I am sorry if you read it like that.

I also know that you are busy, since the training is starting tomorrow.

I am also not *demanding* anything.

Actually I consider as an insult !, Sorry but yes !

Taking it a step further (I am not saying you have to, but usually, a line break is not that time consuming anyway), just saying, still I am not demanding ....

Since I do not like getting insulted and since it seems for me me that you are not interested in improving the writing/wording (everything besides the technical) part of the trainings I will do not anything at all related to the training docs anymore, this goes for this repository as for the website as for hosting, travis, etc.

If you need something related to that, please get in touch with the AI-Team.

Thanks !

Reply:
We're working on the trainings since we have to give it tomorrow. I agree with many things you write but there is a time for rewriting chapters and adding line-breaks and this is not it. It's 100% ok to push small fixes to master. Please do not use a simple fix as a excuse to demand changes or for rants about bad quality. 
I'm very happy with the state that the trainings are in even if there is room for improvement.